src/prepare-build.ts

```diff
@@ -291,8 +293,19 @@ async function getPreviousVersionInfo(shortName: string, publishDate: string) {
 
 		const thisDate = thisURI.match(/[1-2][0-9]{7}/)![0];
```

Comment:
Not sure if it's related but I'm getting `SyntaxError: Unexpected token '!'`
```suggestion
		const thisDate = thisURI.match(/[1-2][0-9]{7}/)[0];
```

Reply:
It's TypeScript; needs to be compiled (`npm run build` or `yarn build`).

Reply:
Sorry, I was comparing the code with spec-generator and didn't pay attention to that.